lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -242,6 +242,11 @@
             // Install event handlers
             $( "#webcam_reset_button", this.el ).on( 'click', _.bind( this.reset, this ) );
             $( "#webcam_capture_button", this.el ).on( 'click', _.bind( this.capture, this ) );
+            $("#webcam_capture_button").keyup(function(event){
+                if(event.keyCode == 13){
+                        $("#webcam_capture_button").click();
+                    }
+                });
```

Comment:
@wedaly  event.keyCode is standard, although I would question why the capture button is calling .click on the element as opposed to just calling this.capture(event)
It could also make sense to have a util function/object that may the keyCode numbers to more meaningful names. e.g.
```
var key = {
    enter: 13,
};
```